compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -611,6 +625,15 @@ def _event_loop(self):
             except (InvalidCommandError, InvalidUserError) as e:
                 exc_type = type(e).__name__
                 log.error(f"({exc_type}) {e}{identity_for_log}")
+                msg = (
+                    f"({exc_type}) unexpected error; this is due to either an endpoint"
```

Comment:
Like the extra debugging msg mechanism

Reply:
The "human, go make a bug report" directive?  Heh.  Thanks.